lib/front-end-builds-notifier.js

```diff
@@ -30,53 +30,51 @@ module.exports = CoreObject.extend({
     return { sha, branch };
   },
 
-  notify() {
+  async notify() {
     const html = this.getIndexContent();
     const { branch, sha } = this.gitInfo();
     const signature = this.sign(html);
     const app_name = this.readConfig('app');
     const data = { app_name, branch, sha, signature, html };
     const plugin = this;
+    const endpoint = plugin.readConfig('endpoint');
+    const febEndpoint = endpoint.match(/\/front_end_builds\/builds$/)
+      ? endpoint
+      : endpoint + '/front_end_builds/builds';
+    const requestOptions = {
+      method: 'POST',
+      uri: febEndpoint,
+      form: data,
+      ...plugin.readConfig('requestOptions'),
```

Comment:
no idea what this config is but is it required/compatible with `node-fetch`?

Reply:
It originally allowed apps to supply extra configuration options to the [`request`](https://github.com/request/request) library (that was used by the addon before `node-fetch`). We don't use this, but leaving it in for the sake of compatibility.

Although, any apps would likely need to adjust the configuration options to be compatible with `node-fetch` rather than `request`, so it's probably not that useful 🤔.